app/importers/file_importers/educators_importer.rb

```diff
@@ -18,10 +18,11 @@ def school_ids_dictionary
 
   def import_row(row)
     educator = EducatorRow.new(row, school_ids_dictionary).build
+    searchbar = PrecomputeSearchbarJson.new
 
     if educator.present?
       educator.save!
-      educator.save_student_searchbar_json
+      searchbar.for(educator)
```

Comment:
This actually needs to be a separate after_imports item. Calculating which students are accessible for an educator takes into account sections. Those are imported after the educators have been imported. 

Seems like the proper place for this would be import.thor after all the imports have completed.

Reply:
Agree!  The searchbar isn't about importing, it's about precomputing an index for a feature.